content/_index.html

```diff
@@ -263,17 +263,19 @@
 	}(document, 'script', 'facebook-jssdk'));</script>
 <section>
 	<header>
-		<h1>Turning Hearts</h1>
+		<!-- <h1>Turning Hearts</h1> -->
+		<p></p>
 		<!-- <h2>Subtitle</h2> -->
 	</header>
 	<!-- <p>The time is short and there are limited places to stay, so please register early!</p> -->
 	<!-- <p data-after="2023-01-01 GMT-0700" data-before="2023-08-02T06:00-0700">We will attempt to stream our meetings live on our <a href="https://www.youtube.com/channel/UCAinceLJDt98CJWP4hbzn2A">YouTube channel</a> and here on this site, however the availability of internet access at the meeting location has not been verified. If we cannot stream live, we will be uploading the recorded video afterwards.</p> -->
 	<!-- <p data-after="2023-08-02T06:00-07:00" data-before="2023-08-07T11:00-07:00">Our apologies, the location's Internet access was unable to support live streaming. We will be posting the recordings on our <a href="https://www.youtube.com/channel/UCAinceLJDt98CJWP4hbzn2A">YouTube channel</a>. Please subscribe to the <a href="https://www.youtube.com/channel/UCAinceLJDt98CJWP4hbzn2A">channel</a> and click the bell icon to be notified.</p> -->
-	<!-- <div>
+	<div>
 		<div class="video">
-			<iframe src="https://www.youtube.com/embed/TNy-Xc3fgbI?list=PLqVg8iJjuJ4fscmvdzrJjDr_7iAl1ElWy&rel=0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+			<!-- <iframe src="https://www.youtube.com/embed/TNy-Xc3fgbI?list=PLqVg8iJjuJ4fscmvdzrJjDr_7iAl1ElWy&rel=0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe> -->
+			<img src="/img/2024banner.png" >
```

Comment:
It's always best to have a `title` and `alt` attributes for images so that those who cannot see but can still hear what is being described via a screen reader. `alt` is more important than `title`.